components/proxy/conf/Caddyfile

```diff
@@ -186,6 +186,27 @@ https://{$GITPOD_DOMAIN} {
 	}
 }
 
+https://payment.{$GITPOD_DOMAIN} {
```

Comment:
@AlexTugarev please move the block to https://github.com/gitpod-io/gitpod/blob/main/chart/templates/proxy-configmap.yaml#L16 (not sure payment is always enabled or not)

Reply:
fixed